src/core/Akka.API.Tests/CoreAPISpec.ApproveStreams.approved.txt

```diff
@@ -1528,17 +1528,21 @@ namespace Akka.Streams.Dsl
     }
     public class static RestartFlow
     {
-        public static Akka.Streams.Dsl.Flow<TIn, TOut, Akka.NotUsed> OnFailuresWithBackoff<TIn, TOut, TMat>(System.Func<Akka.Streams.Dsl.Flow<TIn, TOut, TMat>> flowFactory, System.TimeSpan minBackoff, System.TimeSpan maxBackoff, double randomFactor) { }
+        public static Akka.Streams.Dsl.Flow<TIn, TOut, Akka.NotUsed> OnFailuresWithBackoff<TIn, TOut, TMat>(System.Func<Akka.Streams.Dsl.Flow<TIn, TOut, TMat>> flowFactory, System.TimeSpan minBackoff, System.TimeSpan maxBackoff, double randomFactor, int maxRestarts) { }
         public static Akka.Streams.Dsl.Flow<TIn, TOut, Akka.NotUsed> WithBackoff<TIn, TOut, TMat>(System.Func<Akka.Streams.Dsl.Flow<TIn, TOut, TMat>> flowFactory, System.TimeSpan minBackoff, System.TimeSpan maxBackoff, double randomFactor) { }
```

Comment:
The JVM doesn't have an overload. Should we just make a breaking change here in order to keep the API in sync?

Reply:
@ismaelhamed IMHO, no - I think we should just add a new method. Better to not break the API if we can help it. Would you mind making that change?

Reply:
Done.